### ISSUE TITLE
Add ability to hide back button on event type booking page

### ISSUE THREAD

**Issue by RobertWSaunders** [calcom/cal.com#5056]
### Describe the solution you'd like

We send our users booking links to specific event types, when we ask them to schedule a time, we don't want them to be able to see our Cal.com homepage with all the other public event types:

![CleanShot 2022-10-17 at 15 47 44@2x](https://user-images.githubusercontent.com/13367451/196269167-5158c388-97fc-4b1e-a38a-87ee465999fc.png)

### Describe alternatives you've considered

Right now we hide all of our event types so when they go back they see an ugly empty page:

<img width="1383" alt="CleanShot 2022-10-17 at 15 50 48@2x" src="https://user-images.githubusercontent.com/13367451/196269840-25978319-0b25-4396-b223-28de838ac8a0.png">

**Comment by shirazdole** [comment]
cc @Jaibles 
https://github.com/calcom/cal.com/issues/5048#issuecomment-1282398696

**Comment by PeerRich** [comment]
@Jaibles @sean-brydon do we wanna hide the button if a user has no public event-types?

**Comment by PeerRich** [comment]
regardless, this is a low priority ticket

**Comment by Jaibles** [comment]
Yes ideally we would hide the button if a user has no public event types.
